### ISSUE TITLE
Remove CSS import from core root export?

### ISSUE THREAD

**Issue by monospaced** [measuredco/puck#319]
Having `import "./styles/global.css"` in [/packages/core/index.ts](https://github.com/measuredco/puck/blob/main/packages/core/index.ts#L1) causes a side effect of importing all the puck CSS — including `https://rsms.me/inter/inter.css` and the Inter font(s) — if _any_ of the exports in the file are used.

So if a user imports anything at all from `@/core` (including types, hooks and helpers) they will get _all_ of the puck CSS (and the Inter files) bundled in their app. And unless they are conditionally importing in `/edit` routes onlly, these styles and fonts will also end up in the user's published pages.

I guess the CSS import is there because the index file also exports some UI components (`Button`, `Drawer`, etc.), which need the styles to work. But in the demo app and the next recipe, we explicitly import the puck styles with `import "@/core/styles.css";`. So i think this is a better pattern – you must explicilty import the styles.

Some users may be replying on this import, so removing it would be a breaking change.

It would also be good to update the demo and next reciepe so that Inter and the puck CSS aren't bundled into the published pages by default. Currently we're crossing the streams, because the demo actually _uses_ Inter, and even some of the puck components (e.g. `Button`) for convenience. But in real-world use cases, you’d mostly likely _not_ want the puck CSS and Inter bundled into production.

**Comment by chrisvxd** [comment]
Thanks @monospaced!

My hunch is that you're noticing this issue whilst developing applications in the monorepo rather than external applications. Is that right?

In that case, there is a difference between the import behaviour between the distributed package and monorepo development using the `@` path helpers.

The distributed package (i.e. `@measured/puck`) _should not_ import any CSS when importing types etc, because the files are completely separate and we don't know if the host application includes CSS modules. That's why you have to do `@import "@measured/puck/puck.css";`.

Importing the codebase locally (via the `@` helpers, i.e. `@/core`) _will_ import the CSS, but that's only because the local applications (demo / docs) themselves implement CSS modules and will be detecting the CSS imports and loading them.

**Comment by monospaced** [comment]
@chrisvxd yep in monorepo. I will double check using the distributed package for sanity 👍🏻

**Comment by chrisvxd** [comment]
@monospaced can this be closed?

**Comment by monospaced** [comment]
@chrisvxd I was just about to put in a PR that update the recipe(s) to close this. 

As the recipe is including the CSS in the layout (like `measuredco-site` was), so by default the puck CSS pollutes the published pages.
